bcbio/variation/vcfutils.py

```diff
@@ -324,6 +330,25 @@ def _do_merge(orig_files, out_file, config, region):
         bgzip_and_index(out_file, config)
     return out_file
 
+def _do_combine_variants(orig_files, out_vcf, ref_file, config, region, minimumN = 3):
+    """combine variants with gatk3 using minimum N threshold for PureCN PON"""
+    gatk3 = config_utils.get_program("gatk3", config)
+    cmd = [gatk3, "-Xmx12g",
+           "-T", "CombineVariants",
+           "-R", ref_file,
+           "-o", out_vcf,
+           "--minimumN", minimumN]
+    out_gz = out_vcf + ".gz"
+    if not os.path.exists(out_gz):
+        try:
+            cmd_line = " ".join([str(x) for x in cmd]) + " -V " + " -V ".join(orig_files)
+            do.run(cmd_line, "combine variants")
+        except subprocess.CalledProcessError as msg:
+              logger.info("PON merge failed")
```

Comment:
Does this report the error, or just prints that error message? If that's the case, perhaps reporting it would help. Also, AFAICS, this makes errors go on "semi silently": I'm not sure that's what we want.

There's another instance of this somewhere else, same problem.